<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Library App</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QPushButton" name="btnMember">
                        <property name="text">
                            <string>Member</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnStaff">
                        <property name="text">
                            <string>Staff</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnBook">
                        <property name="text">
                            <string>Book</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="btnProductTable">
                        <property name="text">
                            <string>Product Table</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar" />
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>25</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
            </widget>
            <addaction name="menuFile"/>
            <widget class="QMenu" name="menuProduct">
                <property name="title">
                    <string>Product</string>
                </property>
            </widget>
        </widget>
        <widget class="QToolBar" name="toolBar">
            <property name="windowTitle">
                <string>toolBar</string>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>579</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>